### DISCUSSION TITLE
What's the best practice in Uno to display an image?

### DISCUSSION THREAD

**Discussion by ForkBug** [unoplatform/uno#18190] [Q&A] [answered]
In maui, the best practice to display an image is using FFImageLoading, which provides lazy loading, cache, and automatic downsampling.

In Uno, ImageEx and ImageCache from Windows Community Toolkit provide lazy loading and cache. But no automatic downsampling.

What's the best practice in Uno to show an image?

**Comment by ForkBug** [comment]
Automatic downsampling is useful when a 900x900 image is loaded into a 30x30 image control.

**Reply by MartinZikmund** [reply, marked as answer]
Indeed, we don't currently have clear guidance for this and it would be valuable. I will create a separate issue to track this. In general, all .NET libraries including FFImageLoading should be supported in Uno Platform, so feel free to test this against Uno as well.

**Reply by MartinZikmund** [reply]
Issue #18225 was created to track this

**Comment by ForkBug** [comment]
Sorry, ImageEx and ImageCache seem to be deprecated in  Windows Community Toolkit 8.0
